File: readability/readability.c
#include <cs50.h>
#include <ctype.h>
#include <math.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    string paragraph = get_string("Text: ");
    int letters = 0;
    int words = 1;
    int sentences = 0;

    int n = strlen(paragraph);

    for (int i = 0; i < n; i++ )
    {
        if (isalpha(paragraph[i]))
        {
            letters +=1;
        }

        if (paragraph[i] == ' ')
        {
            words +=1;
        }

        if (paragraph[i] == '.' || paragraph[i] == '!' || paragraph[i] == '?' )
        {
            sentences +=1;
        }

    }

    float L = (float) letters/words*100;
    float S = (float) sentences/words*100;
    int index = round(0.0588 * L - 0.296 * S - 15.8);

    if (index <= 16 && index > 1)
    {
        printf("Grade %d\n", index);
    }
    else if (index >16)
    {
        printf("Grade 16+\n");
    }
    else
    {
        printf("Before Grade 1\n");
    }

}

File: sentimental-readability/readability.py
text = input("Text: ")
letters = 0
words = 1
sentences = 0

for i in range(len(text)):
    if text[i].isalpha():
        letters+=1
    if text[i]==' ':
        words+=1
    if text[i] == '.' or text[i] == '?' or text[i] == '!':
        sentences+=1


L = float(letters/words*100)
S = float(sentences/words*100)
index = round(0.0588 * L - 0.296 * S - 15.8)

if index <= 16 and index > 1:
    print(f"Grade {index}")
elif index > 16:
    print("Grade 16+")
else:
    print("Before Grade 1")
